client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,13 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink to = {`/movies/${movie.id}`} >
```

Comment:
Awesome work adding in the NavLink! Now you can try styling it with the `activeStyle` or `activeClassName` prop!